go.mod

```diff
@@ -86,11 +86,11 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
 	helm.sh/helm/v3 v3.7.2
-	k8s.io/api v0.22.6
-	k8s.io/apiextensions-apiserver v0.22.6
-	k8s.io/apimachinery v0.22.6
+	k8s.io/api v0.23.0
+	k8s.io/apiextensions-apiserver v0.23.1
+	k8s.io/apimachinery v0.23.1
 	k8s.io/cli-runtime v0.22.6
-	k8s.io/client-go v0.22.6
+	k8s.io/client-go v0.23.0
```

Comment:
As I mentioned in one of my PRs, it is just an automatic update when running `go mod tidy` (or similar) commands. However, given that we have fixed the version to 0.22.6 on the `replace` section, the effective dependency is not being upgraded though.

Reply:
not sure if you're asking me to do something here?